### PR TITLE
Add CONTRIBUTING.md [fixes #346]

### LINKED ISSUE #346: Add CONTRIBUTING.md
Two audiences:

- Users who want to file an issue
- Devs
    - Code style
    - Pull requests & tags

This is a subtask of #261.

### PR DESCRIPTION
**Pull request by jeremy-w**
Also makes progress towards #261.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,247 @@
...
+  - Alternatively, if you're comfortable with the Terminal, run
+    `sw_vers` and paste the output.
+
+- If it's Linux,
+  provide information on your distro and any customization
+  relevant to Plover's operation (desktop environment, window manager, ???).
+  - uname -a output might also be useful.
+
+- If it's Windows,
+  (Ted/Benoit: Where should someone look for the Windows version?)
```
Comment (jeremy-w) [inline CONTRIBUTING.md]: @openstenoproject/developers I could use your help on filling out the "where to find the Windows version" section of this. :)

**Comment by morinted** [comment]
This is super great! Thanks so much for the work!

I'm heading off to bed so this is coming from my phone... Will be brief.

- Windows version can be seen with Windown key + Pause Break.
- Should maybe state that users can't set their own tags, and that they are set by OSP members.
- Remove my casual language in regard to wish list, maybe.

This PR is really great, thanks!

Finally, if I had to nitpick, I'd s/till/'til or until.

**Comment by jeremy-w** [comment]
@morinted I don't think I actually understand the difference in meaning between feature-request and wishlist. Could you clarify that for me?

**Comment by morinted** [comment]
Basically wishlist is an accepted feature request: we want to work on it.
Anyone can make a feature request, but we need to evaluate it before
putting it on the wishlist.

**Comment by jeremy-w** [comment]
@morinted Please review the latest changes and merge if all's well.

**Pull request merged**